app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/masterpasswordguidance/MasterPasswordGuidanceScreen.kt

```diff
@@ -81,126 +78,134 @@ fun MasterPasswordGuidanceScreen(
+        MasterPasswordGuidanceContent(
             modifier = Modifier
                 .fillMaxSize()
                 .verticalScroll(rememberScrollState())
                 .standardHorizontalMargin(),
-        ) {
-            Column(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .clip(RoundedCornerShape(size = 4.dp))
-                    .background(BitwardenTheme.colorScheme.background.tertiary),
-            ) {
-                Column(
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .padding(all = 24.dp),
-                ) {
-
-                    Text(
-                        text = stringResource(R.string.what_makes_a_password_strong),
-                        style = BitwardenTheme.typography.titleMedium,
-                        color = BitwardenTheme.colorScheme.text.primary,
-                    )
-                    Spacer(modifier = Modifier.height(8.dp))
-                    Text(
-                        style = BitwardenTheme.typography.bodyMedium,
-                        color = BitwardenTheme.colorScheme.text.primary,
-                        text = stringResource(
-                            R.string.the_longer_your_password_the_more_difficult_to_hack,
-                        ),
-                    )
-                }
-                BitwardenHorizontalDivider()
-                Column(
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .padding(horizontal = 24.dp, vertical = 16.dp),
-                ) {
-                    Text(
-                        text = stringResource(R.string.the_strongest_passwords_are_usually),
-                        style = BitwardenTheme.typography.titleSmall,
-                        color = BitwardenTheme.colorScheme.text.primary,
-                    )
-                    Spacer(modifier = Modifier.height(8.dp))
-                    BulletTextRow(text = stringResource(R.string.twelve_or_more_characters))
-                    BulletTextRow(
-                        text = stringResource(
-                            R.string.random_and_complex_using_numbers_and_special_characters,
-                        ),
-                    )
-                    BulletTextRow(
-                        text = stringResource(R.string.totally_different_from_your_other_passwords),
-                    )
-                }
-            }
-            Spacer(modifier = Modifier.height(16.dp))
-            TryGeneratorCard(
-                onCardClicked = remember(viewModel) {
-                    {
-                        viewModel.trySendAction(
-                            MasterPasswordGuidanceAction.TryPasswordGeneratorAction,
-                        )
-                    }
-                },
-            )
-            Spacer(modifier = Modifier.navigationBarsPadding())
-        }
+            viewModel = viewModel,
```

Comment:
Don't pass the whole VM into the content, just pass in the state and any lambdas needed.

Reply:
okay sounds good!  Just curious, is there a downside to passing in the `viewModel` or just a style thing?

Reply:
actually after implementing the change it became much clearer to me why this is a better approach.  Thanks!

Reply:
It can also help with previews and individual composable tests (even if we rarely do those)